### DISCUSSION TITLE
Insufficient privileges to get access token

### DISCUSSION THREAD

**Discussion by felonyf0x** [SteamRE/DepotDownloader#494] [Q&A]
### What did you expect to happen?

the download starts

### Instead of that, what actually happened?

Insufficient privileges to get access token for app

On Powershell:
Logging 'My Steam Login name' into Steam3...
STEAM GUARD! Use the Steam Mobile App to confirm your sign in...
 Done!
Using Steam3 suggested CellID: 25
Got 356 licenses for account!
Insufficient privileges to get access token for app 1778822
App 1778822 () is not available from this account.
Disconnected from Steam
PS C:\Users\Administrator\Downloads\DepotDownloader-windows-x64>

### Which operating system are you running on?

Windows

### Version

2.5.0

### Relevant log output

```shell
Logging 'xbox720' into Steam3...
STEAM GUARD! Use the Steam Mobile App to confirm your sign in...
 Done!
Using Steam3 suggested CellID: 25
Got 356 licenses for account!
Insufficient privileges to get access token for app 1778822
App 1778822 () is not available from this account.
Disconnected from Steam
PS C:\Users\Administrator\Downloads\DepotDownloader-windows-x64>
```

**Comment by felonyf0x** [comment]
the last time i try show exactly this

![image1](https://github.com/SteamRE/DepotDownloader/assets/94789703/8af1b84c-b11c-4ec3-8667-e7facb51a673)

**Comment by yaakov-h** [comment]
Do you own Tekken 8 on that account with the censored username?

**Reply by felonyf0x** [reply]
hi yaakov, aparently i put wrong numbers in the line, coul'd you please check for me, if now my command line is ok?

.\depotDownloader.exe -app 1778820 -depot 1778821 -manifest 6933485284773485049 (now it's downloading but...)

![image](https://github.com/SteamRE/DepotDownloader/assets/94789703/2b829ee9-4273-42d3-804a-75ad145ce964)

My intention is to only download the previous update. It seems that with this command line, I'm downloading the entire game. Could you please provide me with the correct command line to download only the update? Usually, updates are around 1 to 2 gigabytes, which is a significant difference compared to the total size of the game, which is 80 GB. Thank you! I'm not very fluent in English, so I'm using ChatGPT.

**Reply by SlejmUr** [reply]
There is no command line argument to only download the update. Either full files or nothing

**Reply by felonyf0x** [reply]
not problem!! for all the support thanks! have a nice weekind.

**Comment by felonyf0x** [comment]
hi man, first, thanks to respond, yes i buy, first week.
 
 



De: "Yaakov" ***@***.***>
Enviada: 2024/04/24 09:46:20
Para: ***@***.***
Cc:  ***@***.***, ***@***.***
Assunto:  Re: [SteamRE/DepotDownloader] Insufficient privileges to get access token (Issue #489)
 

 

Do you own Tekken 8 on that account with the censored username?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/SteamRE/DepotDownloader/issues/489#issuecomment-2074865801", "url": "https://github.com/SteamRE/DepotDownloader/issues/489#issuecomment-2074865801", "name": "View Issue" }, "description": "View this Issue on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by felonyf0x** [comment]
if you coul'd check with someone haves the game too, will be nice, i'm trying to find someone to do the same, i don't know if no works because my version is Latin.

**Reply by felonyf0x** [reply]
hi, i try the new version of Depot, but even with Dotnet8, the same error continues.
